playbooks/adhoc/uninstall.yml

```diff
@@ -85,7 +85,7 @@
 
   - name: Remove packages
     package: name={{ item }} state=absent
-    when: not is_atomic | bool
+    when: not is_atomic | bool and openshift_remove_all | default(True) | bool
```

Comment:
The example I gave above had nested blocks, so the inside when is applied to the innermost block.